t/129-ssl-socket.t

```diff
@@ -2834,3 +2829,66 @@ SSL reused session
 [alert]
 [emerg]
 --- timeout: 10
+
+
+
+=== TEST 35: www.google.com in init_worker_by_lua
+--- http_config
+    init_worker_by_lua_block {
```

Comment:
I'm afraid it's not a good idea to block init worker hook for long. Because new requests are already coming in. Blocking for long in this phase can be problematic. We need to make it very clear in the documentation at least. Honestly we should really disable such blocking emulations in this init worker context.